torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -266,10 +261,9 @@ def _fill_tensor_shape_type(
         else:
             # We set node output sizes to be dynamic to continue the model conversion,
             # and inputs are also set to be dynamic in add_input().
-            onnxscript_value.shape = tuple(
-                [dim if isinstance(dim, int) else None for dim in expected_value.size()]
-            )
+            onnxscript_value.shape = expected_value.size()
```

Comment:
I recall we avoid introducing FakeTensor concept to onnxscript, but here, we are shiping torchSym family concept to onnxscript. Is it because torch.Sym is public, while fake tensor are not?

Reply:
I wrote it like this to keep the sym handling logic in one place.